### DISCUSSION TITLE
Welcome to YimMenu Discussions!

### DISCUSSION THREAD

**Discussion by Yimura** [YimMenu/YimMenu#136] [Announcements]
I have enabled discussions as an _experiment_, I have never used it before but with the recent issues that have gone off-topic.

These issues contain a lot of information and it would be a shame to put an end to such things being shared.

Feel free to share any research you've done on GTA V, discuss the future of YimMenu, what you would like done differently, etc...

**Comment by JustSulo** [comment]
Ok so i downloaded the Menu and had some fun on GTA V, so there is yet no feature to add money/drop money or change your RP? Im new at this and im confused xD

**Reply by Rxann** [reply]
YimMenu focuses on non money or rp related features so it won't be "targeted" by rockstar.

**Comment by nane24685** [comment]
Im not skilled in these comp languages i dont even know is this the way to ask a question but ill try, i have used yimmenu for a half year recently my system fell down i lost everything no i cant download xenos extreme injector not even one from all of them ,can someone please help me ,yimmenu best menu ever i wnat it back i need it i cant play at all someone PLEASE HELP ME

**Reply by pierrelasse** [reply]
I recommend you could make a new discussion for that.

Now what happened is:
- You lost all your data?
- Now got your pc working again?
- Can't download xenos or extreme injector for some reason? (maybe try not using internet explorer jk)

Or did i miss something?
Can you download other files from this internet thingy aka. WWW?

**Comment by Tessio** [comment]
Are there any future plans to change the UI or improve it?

Maybe a YimMenu v2 Remake 🔥

**Reply by pierrelasse** [reply]
YimMenu V2 will be in GTA6 ig xD

**Reply by YimMenuTheBest** [reply]
A little polish would be nice, maybe make it look a little more modern and user friendly. 

This is what I would like to see in the future.

Typical necessary features:

- Being able to move the menu position.
- Turn the Tuning and Outfit Editor sections into Windows, to avoid not being able to see what you are doing.
- Re work on the Notify system, those notifications could be nicer by applying a sound and a time bar-label when they disappear.

And an extra:
- Command Bindings

**Reply by pierrelasse** [reply]
Yeah i also thought of making a feature request or related about the notifications.

Even in the Horse Menu they are better
![image](https://github.com/YimMenu/YimMenu/assets/76223390/592196d5-37c5-428a-84de-9d47bdbed4e8)

Here are also other suggestions for the hotkeys https://github.com/YimMenu/YimMenu/discussions/2402

And a offical discord would be cool to publish scripts etc etc

**Reply by Tessio** [reply]
Completely necessary

**Reply by Tessio** [reply]
They are much better.
and a discord server should also be planned for social relations, shares, etc.

**Reply by pierrelasse** [reply]
https://github.com/YimMenu/YimMenu/discussions/2224 this was meant to go in the non existing #memes channel but the only thing it was was cringe...........

**Reply by Yimura** [reply]
I agree YimMenu may be lacking in several parts and compared to HorseMenu in particular the notifications it looks pretty bad. But you've got to understand that HorseMenu started development 2 years after YimMenu went public and is thus able to adapt more easily to the pain-points YimMenu has.

In the a lot these things are easier said than done.
Rebuilding the menu from scratch would take a very very long time, even if you're still just copy pasting a lot of stuff.

**Comment by YimMenuTheBest** [comment]
I don't mean Recovery Money, but adding a collectibles unlocker would be nice.

If we have the Goosh event, why not add the Trigger Scripts of the collectibles?

**Reply by pierrelasse** [reply]
I think that they don't want recovery and unlock options in the base menu. Maybe a official script repo inside the menu like stand would be cool (with downloads counted etc etc etc blah blah).

**Reply by Tessio** [reply]
I think you have a point, but as @pierrelasse says we could create an official community script where we complement a never-before-seen unlocker....

**Reply by YimMenuTheBest** [reply]
I like the 2 ideas, I hope to carry it out, I have a lot of things to share.

**Reply by pierrelasse** [reply]
Github is a bit more like emails (better than emails but still i hope you get the point).

Why not make an unofficial discord server? :)))

**Reply by Yimura** [reply]
You're free to do whatever, however remember that starting a Discord server for YimMenu comes with the likelihood of getting the server owner's account terminated by Discord staff and possibly also any other people that are moderating the Discord server. Combined with the fact that you'll have to moderate content as well, you'll need to look out for malicious binaries being distributed and possibly getting guild members compromised.

Very important point I want to get across is do not act as if you're official in any shape or farm, make it obvious that you're unofficial and do not get power hungry just because you're in a position of power.

**Reply by pierrelasse** [reply]
Stupid discord :/ sadly no good alternatives
